py-json/station_profile.py

```diff
@@ -229,12 +280,12 @@ def set_reset_extra(self, reset_port_enable=False, test_duration=0, reset_port_m
     #   - 'wpa3'
     def use_security(self, security_type, ssid=None, passwd=None):
         SECURITY_TYPES = {
-            "open":    "[BLANK]",
```

Comment:
In the future, do not mix whitespace changes with logic changes. This type of change would be better included as a separate commit that indicates it only changes whitespace/commenting.

Reply:
No change required here